src/FormBuilder/resources/js/form-builder/src/settings/group-donation-confirmation/index.tsx

```diff
@@ -0,0 +1,51 @@
+import {__} from '@wordpress/i18n';
+import {createInterpolateElement} from '@wordpress/element';
+import {SettingsSection} from '@givewp/form-builder-library';
+import DonationConfirmation from './donation-confirmation';
+import TemplateTags from './template-tags';
+import {setFormSettings, useFormState, useFormStateDispatch} from '@givewp/form-builder/stores/form-state';
+
+export default function FormDonationConfirmationSettingsGroup() {
+    const {
+        settings: {receiptHeading, receiptDescription},
+    } = useFormState();
+    const dispatch = useFormStateDispatch();
+
+    const templateTagsDescription = createInterpolateElement(
+        __(
+            'Available template tags for this email. HTML is accepted. <a>See our documentation</a> for examples of how to use custom meta email tags to output additional donor or donation information in your GiveWP emails',
+            'give'
+        ),
```

Comment:
A couple things here:

1. This is not the right text for this setting as these are not _email_ settings
2. Previously there were separate template tags per content setting (heading & description) - are we not doing that anymore?

Reply:
1. I asked @jdghinson to check for the updated copy.
2. In the previous UI we did had separated tags per setting as they used to be editable in a popover. Since they both had the same available tags, the new UI brings them as a single section in the page.

![CleanShot 2024-01-04 at 22 17 16](https://github.com/impress-org/givewp/assets/3921017/4add2b7a-2dbc-418e-9e1a-792bec99b3aa)

Reply:
@jonwaldstein #1 was resolved on 8e444e478c0e7908401bae5fba3875486b443c44